content/docs/command-reference/exp/pull.md

```diff
@@ -6,13 +6,13 @@ data from a `dvc remote`.
 ## Synopsis
 
 ```usage
-usage: dvc exp pull [-h] [-q | -v] [-f] [--no-cache]
-                    [-r <name>] [-j <number>] [--run-cache]
+usage: dvc exp pull [-h] [-q | -v] [-A] [--rev <commit>] [-n <num>] [-f]
+                    [--no-cache] [-r <name>] [-j <number>] [--run-cache]
                     git_remote experiment
 
 positional arguments:
   git_remote          Git remote name or Git URL
-  experiment          Experiment to pull
+  experiment          Experiments to pull
```

Comment:
Would prefer to discuss this separately. Seems out of scope for this already long-standing PR.
```suggestion
  experiment          Experiment to pull
```